pkg/gadgettracermanager/containers-map.h

```diff
@@ -12,13 +12,4 @@ struct {
 	__uint(max_entries, MAX_CONTAINERS_PER_NODE);
 } containers SEC(".maps");
 
-#ifdef WITH_FILTER
-struct {
-	__uint(type, BPF_MAP_TYPE_HASH);
-	__type(key, __u64);
-	__type(value, __u32);
-	__uint(max_entries, MAX_CONTAINERS_PER_NODE);
-} mount_ns_filter SEC(".maps");
```

Comment:
Do you mean that we should keep the map in the header file and remove from all *.bpf.c? I was wondering the same, perhaps we can find a way to factorize the filter by mount ns idea later on? (like having some helpers in a common place to avoid writing the same lines again and again there).